mmv1/products/compute/terraform.yaml

```diff
@@ -2053,6 +2053,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           region_url_map_name: "regionurlmap"
           home_region_backend_service_name: "home"
           region_health_check_name: "health-check"
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
Do you want to add a `skip_test: true` to this example for now, and then I can remove it as a part of https://github.com/hashicorp/terraform-provider-google/issues/10202 ? 

Just so we can get the sample generated?

Reply:
Done